examples/async-examples.ts

```diff
@@ -0,0 +1,53 @@
+import {RevAiApiClient, RevAiJobOptions} from 'revai-node-sdk';
```

Comment:
minor -
```suggestion
import { RevAiApiClient, RevAiJobOptions } from 'revai-node-sdk';
```

Reply:
imports are being restructured as the examples will be `.js` rather than `.ts`